server/app/controllers/NewTask.scala

```diff
@@ -39,13 +43,16 @@ class NewTask @Inject()(executor: DynamicSuiteExecutor, dao: Dao, val messagesAp
       f => {
         val futureResponse = for {
           _ <- Future(StringBuilderRunner(executor(f.referenceSolution, f.suite))).check
-          db <- dao.addTask(Task(scalaClass, f.taskDescription, f.solutionTemplate, f.referenceSolution, f.suite))
+          db <- dao.addTask(
+            Task(new Timestamp(System.currentTimeMillis),
```

Comment:
We will put as Int when you will prove that it is better, I will agree :) not a big deal